### PR TITLE
fix: when no input enableECSManagedTagsInput, not include it to request params

### PR DESCRIPTION
**Pull request by wim-web**
*Description of changes:*

If enableECSManagedTags is set to true by IaC, such as Terraform, and enable-ecs-managed-tags is not specified in the input for this action, each time the action is run, enableECSManagedTags will rollback to false.

you would think it is good to specify enable-ecs-managed-tags, but it is need to modify both code.

So modified the code to leave it to the infrastructure side setting unless explicitly specify it in action.

ref: #622

### CODE REVIEW

**Comment by wim-web** [comment]
@kg-aws

**Comment by wim-web** [comment]
thanks for response @amazreech

fixed it by commiting result executed npm run package!

**Review by omkhegde — APPROVED** [review]

**Review by amazreech — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -398,7 +401,7 @@ async function run() {
     const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || 'false';
     const forceNewDeployment = forceNewDeployInput.toLowerCase() === 'true';
     const desiredCount = parseInt((core.getInput('desired-count', {required: false})));
-    const enableECSManagedTagsInput = core.getInput('enable-ecs-managed-tags', { required: false }) || 'false';
+    const enableECSManagedTagsInput = core.getInput('enable-ecs-managed-tags', { required: false }) || '';
     const enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
```
Comment (karanbokil) [inline index.js:405]: This is still being used in `runTask` - can we include the changes for the enable ecs managed tags behavior in `runTask` as well to keep the behavior consistent?

Comment (karanbokil) [reply index.js:405]: In fact we can just move
```
  enableECSManagedTags = ""
  if (enableECSManagedTagsInput !== "") {
   enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
  }
```
here and set the param in `updateEcsService` and `runTask` only if it's not set to ""

File: index.test.js
Comment (karanbokil) [inline index.test.js:1]: We also still want unit test coverage of setting the property to `true`

**Review by karanbokil — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (wim-web) [reply index.js]: when enableECSManagedTags = “”, ecs.updateService(params) get an error "SerializationException: Unexpected token received from parser"
so enableECSManagedTags is set to null by default

I actually checked with the following code in my aws

```
const ecs = new ECS();
  
let params = {
    cluster: "cluster-name",
    service: "servicepname",
    taskDefinition: "arn:aws:ecs:us-east-1:000000000:task-definition/some-task:1",
    enableECSManagedTags: "", // parse error
    // enableECSManagedTags: null, // ok
    desiredCount: 1,
};

await ecs.updateService(params);
```

**Comment by wim-web** [comment]
@karanbokil 
tried to fix it, and add test

File: index.js (hunk 1 above)
Comment (karanbokil) [reply index.js]: Great find!

**Review by karanbokil — APPROVED** [review]
LGTM!

**Comment by wim-web** [comment]
thank you for review
what should i do next?

**Comment by karanbokil** [comment]
Change is merged! It should be integrated in the next release this month, but till then you can also configure your action to track HEAD to use the feature early

**Pull request merged**